orttraining/orttraining/core/graph/gradient_schema_defs.cc

```diff
@@ -1300,32 +1326,40 @@ Return true if all elements are true and false otherwise.
       .SinceVersion(1)
       .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
       .SetDoc("Send data tensor to the specified destination.")
-      .Input(0, "InputSignal", "Input control signal.", "TBool")
-      .Input(1, "Data", "Tensor to send.", "T")
-      .Output(0, "OutputSignal", "Output control signal.", "TBool")
-      .Attr("src",
-            "Abstractive memory ID of Data's source.",
-            AttributeProto::INT)
-      .Attr("dst",
-            "Abstractive memory ID of Data's destination.",
-            AttributeProto::INT)
+      .Input(0, "Remote", "Remote dst rank. It must be a scalar.", "TInt64")
+      .Input(1, "InputSignal", "Input control signal. It must be a scalar.", "TBool")
```

Comment:
nit: I like to have InputSignal as the first input because OutputSignal is the first output.

Reply:
Fine, I actually changed it from there a few comments before.
I don't mind changing it back.